lib/views/widgets/navigation/navigation_button.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+import 'package:sun_flutter_capstone/consts/global_style.dart';
+
+// TODO: change widget to accept onPressed event
+class NavigationButton extends StatelessWidget {
+  final IconData icon;
+  final Color iconColor;
+
+  const NavigationButton(
+      {Key? key, required this.icon, this.iconColor = AppColor.lightGray})
+      : super(key: key);
```

Comment:
Can I confirm is this the correct indentation here?

Reply:
i think it be changed to multiline for easier readability miss, will change this..